behandlingsprosess/src/main/java/no/nav/foreldrepenger/behandling/steg/dekningsgrad/DekningsgradSteg.java

```diff
@@ -0,0 +1,58 @@
+package no.nav.foreldrepenger.behandling.steg.dekningsgrad;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+import no.nav.foreldrepenger.behandling.EndreDekningsgradVedDødTjeneste;
+import no.nav.foreldrepenger.behandling.FagsakRelasjonTjeneste;
+import no.nav.foreldrepenger.behandlingskontroll.BehandleStegResultat;
+import no.nav.foreldrepenger.behandlingskontroll.BehandlingSteg;
+import no.nav.foreldrepenger.behandlingskontroll.BehandlingStegModell;
+import no.nav.foreldrepenger.behandlingskontroll.BehandlingStegRef;
+import no.nav.foreldrepenger.behandlingskontroll.BehandlingTypeRef;
+import no.nav.foreldrepenger.behandlingskontroll.BehandlingskontrollKontekst;
+import no.nav.foreldrepenger.behandlingslager.behandling.BehandlingStegType;
+import no.nav.foreldrepenger.familiehendelse.FamilieHendelseTjeneste;
+
+@BehandlingStegRef(BehandlingStegType.DEKNINGSGRAD)
+@BehandlingTypeRef
+@ApplicationScoped
+public class DekningsgradSteg implements BehandlingSteg {
+
+    private final FagsakRelasjonTjeneste fagsakRelasjonTjeneste;
+    private final FamilieHendelseTjeneste familieHendelseTjeneste;
+    private final EndreDekningsgradVedDødTjeneste endreDekningsgradVedDødTjeneste;
+
+    @Inject
+    public DekningsgradSteg(FagsakRelasjonTjeneste fagsakRelasjonTjeneste,
+                            FamilieHendelseTjeneste familieHendelseTjeneste,
+                            EndreDekningsgradVedDødTjeneste endreDekningsgradVedDødTjeneste) {
+        this.fagsakRelasjonTjeneste = fagsakRelasjonTjeneste;
+        this.familieHendelseTjeneste = familieHendelseTjeneste;
+        this.endreDekningsgradVedDødTjeneste = endreDekningsgradVedDødTjeneste;
+    }
+
+    @Override
+    public BehandleStegResultat utførSteg(BehandlingskontrollKontekst kontekst) {
+        if (erBarnDødsfall(kontekst)) {
+            endreDekningsgradVedDødTjeneste.endreDekningsgradTil100(kontekst.getBehandlingId());
+        }
+
```

Comment:
Burde man sette YF.sakskompleksDG her ?
Tanken var å ha et steg for evt aksjonspunkt (VP = UT) eller overstyring (neste.INN) slik at beregning rekjøres ved overstyring.

Reply:
Neste steg